extraConfigDpuInfra.py

```diff
@@ -186,7 +186,7 @@ def cb() -> None:
     client.oc("create -f manifests/infra/dpuclusterconfig.yaml")
 
     logger.info("Patching mcp setting maxUnavailable to 2")
-    client.oc("patch mcp dpu --type=json -p=\[\{\"op\":\"replace\",\"path\":\"/spec/maxUnavailable\",\"value\":2\}\]")
+    client.oc("patch mcp dpu --type=json -p=\\[\\{\"op\":\"replace\",\"path\":\"/spec/maxUnavailable\",\"value\":2\\}\\]")
```

Comment:
create a dict and then serialize the json. Same remark for other instances.

Reply:
https://github.com/bn222/cluster-deployment-automation/pull/129